2020-07-07 15:33:53,529: helper | ERROR | gtts Request error
File:    tts.py
Line:    47
Target:  voice_text = recognizer.recognize_google(audio)
Message: recognition request failed: Bad Request
2020-07-07 16:41:56,360: helper | ERROR | gtts Request error
File:    tts.py
Line:    47
Target:  voice_text = recognizer.recognize_google(audio)
Message: recognition request failed: Bad Request
